app/boards/shields/ergodash/ergodash.keymap

```diff
@@ -0,0 +1,95 @@
+&kp  LSHFT    &kp  Z        &kp  X        &kp  C        &kp  V        &kp  B        &kp  END               &kp  HOME     &kp  K        &kp  M        &kp  COMMA    &kp  DOT      &kp  SLASH    &kp  RSHFT
+&kp  LCTRL    /**/ &none    &kp  PG_DN    &kp  LMETA    &kp  LALT     &kp  SPACE    &mo  LOWER             &mo  RAISE    &kp  RET      &kp  RALT     /**/ &none    &kp  PG_UP    &kp  INS      &kp  RCTRL
+                        >;
+                };
+                lower_layer {
+/* .----------------------------------------------------------------------------------------------------------------------.
+ * |  F11 |  F1  |  F2  |  F3  |  F4  |  F5  |      |                    |      |  F6  |  F7  |  F8  |  F9  |  F10 |  F12 |
+ * |------+------+------+------+------+------+------+--------------------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |                    |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+--------------------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |                    |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+---------------------------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |                    |      |      |      |      |      |      |      |
+ * |-------------+------+------+------+------+------+------+------+------+------+------+------+------+------+-------------|
+ * | REST | BOOT |      |      |      |      |      |      |      |      |      |      |      |      |      |      |      |
+ * .----------------------------------------------------------------------------------------------------------------------.
+ */
+                        bindings = <
+&kp  F11      &kp  F1       &kp  F2       &kp  F3       &kp  F4       &kp  F5       &kp  &none             &kp  &none    &kp  F6       &kp  F7       &kp  F8       &kp  F9       &kp  F10      &kp  F12
+/**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none             /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none
+/**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none             /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none
+/**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none             /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none
+&reset        &bootloader   /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none             /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none    /**/ &none
```

Comment:
I appreciate the effort that's gone into adding these comments to line up `&none`, but I'm wondering if they might cause maintenance issues/be annoying for when people customize their keymap.

Reply:
It wasn't really that much effort - removed.